tools/ci_build/github/android/nnapi_supported_ops.md

```diff
@@ -45,6 +45,7 @@ Keep in sync with doco generated from /docs/execution-providers/NNAPI-ExecutionP
 |ai.onnx:Sin||
 |ai.onnx:Slice||
 |ai.onnx:Softmax||
+|ai.onnx:Split|Only even splitting is supported.|
```

Comment:
is it obvious enough what "even splitting" means? maybe "number of splits must evenly divide split axis size"?
should we also mention what inputs need to be constant?

Reply:
okay, updated and added a line for split need to be constant if provided.